pom.xml

```diff
@@ -15,6 +15,8 @@
 	<description>api codeE project </description>
 	<properties>
 		<java.version>17</java.version>
+		<maven.compiler.source>21</maven.compiler.source>
```

Comment:
Is this Java 21? We should really use Java 17 here, else it will break the future CI/CD.